src/librustdoc/config.rs

```diff
@@ -237,9 +237,6 @@ pub(crate) struct RenderOptions {
     pub(crate) default_settings: FxHashMap<String, String>,
     /// If present, suffix added to CSS/JavaScript files when referencing them in generated pages.
     pub(crate) resource_suffix: String,
-    /// Whether to run the static CSS/JavaScript through a minifier when outputting them. `true` by
-    /// default.
-    pub(crate) enable_minification: bool,
```

Comment:
Since it's a stable option, I don't think it can be removed just like that. We need a FCP or at least to deprecate it first. It was very useful for testing rustdoc too but I'm not sure if anyone other than me is using this option, so I \*think\* it's fine to remove it.

Reply:
Then problem solved. :D